
syntax = "proto3";

option java_package = "io.grpc.demo.remoteadddemo";

service RemoteAdd {
    // An RPC for adding two numbers together
    rpc Add(AdditionArgs) returns (AdditionResults) {}
}

message AdditionArgs {
    int32 arg1      = 1;
    int32 arg2      = 2; 
}

message AdditionResults {
    int32 sum       = 1;
}
